### PR TITLE
Data recomm

### PR DESCRIPTION
**Pull request by Shadowsong27**

### CODE REVIEW

File: data/recommedation_algo/recommender.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    def update_user_scale(self):
+        """
+        update the user scale in the database
+        :return:
+        """
+        regressors = []
+        responses = []
+
+        # calculate raw score
+        user_rating_records = self.db.get_user_ratings(self.user_id)  # join public rating remove None
```
Comment (zzzzwen) [inline data/recommedation_algo/recommender.py]: Has default coef implemented for cold start?

File: data/recommedation_algo/recommender.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            if len(result_list) == 10:
+                break
+            user_ratings = self.db.get_user_ratings(self.user_id)
+
+            # get training data to fit model
+            # labelling
+            label = []
+            for user_rating in user_ratings:
+                movie_id, score = user_rating
+                if score >= 8.0:  # consider as favorable movie
```
Comment (zzzzwen) [inline data/recommedation_algo/recommender.py]: 8.0 consider as favorable, explain the logic? What does the if clause do here? I'm assuming that our recommendation is not simply recommended/not recommended right?

File: data/recommedation_algo/recommender.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+            # TODO: quantify available data fields, make logistic regression model
+            # TODO: after classification, iterate through movie pools by year and store recommendation
+
+            break
+
+
+if __name__ == '__main__':
+    warnings.filterwarnings(action="ignore", module="scipy", message="^internal gelsd")  # ignore lapack related warning
+    recommender = Recommender('8')
```
Comment (zzzzwen) [inline data/recommedation_algo/recommender.py]: what does the '8' mean here

File: data/recommedation_algo/recommender.py (hunk 2 above)
Comment (Shadowsong27) [inline data/recommedation_algo/recommender.py]: means select a smaller pool of movies from the 400k

File: data/recommedation_algo/recommender.py (hunk 3 above)
Comment (Shadowsong27) [inline data/recommedation_algo/recommender.py]: testing

**Pull request merged**